src/main/java/spireMapOverhaul/zones/Junkyard/CollectingSpree.java

```diff
@@ -0,0 +1,104 @@
+package spireMapOverhaul.zones.Junkyard;
+
+import basemod.abstracts.events.PhasedEvent;
+import basemod.abstracts.events.phases.TextPhase;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.CardGroup;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.events.AbstractEvent;
+import com.megacrit.cardcrawl.localization.EventStrings;
+import com.megacrit.cardcrawl.rooms.AbstractRoom;
+import com.megacrit.cardcrawl.ui.buttons.LargeDialogOptionButton;
+import com.megacrit.cardcrawl.vfx.UpgradeShineEffect;
+import com.megacrit.cardcrawl.vfx.cardManip.ShowCardAndObtainEffect;
+import com.megacrit.cardcrawl.vfx.cardManip.ShowCardBrieflyEffect;
+import spireMapOverhaul.SpireAnniversary6Mod;
+
+import static spireMapOverhaul.SpireAnniversary6Mod.makeID;
+
+public class CollectingSpree extends PhasedEvent {
+    public static final String ID = makeID("CollectingSpree");
+    private static final EventStrings eventStrings = CardCrawlGame.languagePack.getEventString(ID);
+    private static final String[] DESCRIPTIONS = eventStrings.DESCRIPTIONS;
+    private static final String[] OPTIONS = eventStrings.OPTIONS;
+    private static final String title = eventStrings.NAME;
+    private int chance = 20;
+    private int cardsCollected = 0;
+
+    public static boolean bonusCondition() {
```

Comment:
This does ntohing if you simply return true, no?

Reply:
It didn't work without it, I don't question these things